src/main/java/me/totalfreedom/totalfreedommod/command/Command_tag.java

```diff
@@ -54,11 +54,19 @@ else if ("clearall".equalsIgnoreCase(args[0]))
                 int count = 0;
                 for (final Player player : server.getOnlinePlayers())
                 {
-                    final FPlayer playerdata = plugin.pl.getPlayer(player);
-                    if (playerdata.getTag() != null)
+                    if (!plugin.al.isAdmin(sender))
+                    {
+                        final FPlayer playerdata = plugin.pl.getPlayer(player);
+                        if (playerdata.getTag() != null)
+                        {
+                            count++;
+                            playerdata.setTag(null);
+                        }
+                    }
+                    else
                     {
-                        count++;
-                        playerdata.setTag(null);
+                        // Give the admin their default tag
```

Comment:
You never increment the counter, you probably want a player and admin counter if you were to do something like this.